internal/pkg/flags/flag_to_value.go

```diff
@@ -159,6 +159,17 @@ func FlagWithDefaultToStringValue(p *print.Printer, cmd *cobra.Command, flag str
 	return value
 }
 
+// Returns the bool value set on the flag. If no value is set, returns the flag's default value.
+// Returns false if the flag value can not be converted to bool or if the flag does not exist.
+func FlagWithDefaultToBoolValue(p *print.Printer, cmd *cobra.Command, flag string) bool {
+	value, err := cmd.Flags().GetBool(flag)
+	if err != nil {
+		p.Debug(print.ErrorLevel, "convert flag with default to bool value: %v", err)
+		return false
+	}
+	return value
+}
```

Comment:
This function is exactly the same as `FlagToBoolValue`, right?

Reply:
I don't think we need it, also because bool flags always have a default (they are always either `true` or `false`)